README.md

```diff
@@ -73,3 +73,33 @@ Then run the following command in your bash prompt to download content to your l
 ```bash
 bundle exec jekyll contentful
 ```
+
+### Images
+
+With the help of imgix, we are using two strategies to increase performance while rendering images:
+
+- Pixellated placeholders
+- `srcset` and `sizes` attributes
+
+For more info on how this works, see the extended documentation in the [media repo](https://github.com/crdschurch/crds-media/blob/development/README.md#images).
+
+#### Examples
+
+**Inline Images**
+```html
+<img src="{{ page.image | imgix: site.imgix }}?{{ site.imgix_params.placeholder }}" sizes="{{ site.image_sizes.full_width }}" data-optimize-img>
+```
+
+This would translate to something like the following (before processing):
+
+```html
+<img src="//crds-media-int.imgix.net/5G62zla1zOsmKqSo8wmomI/d46b0ec8a96339c72f25b56b7c2dd99b/isle-of-skye.jpg?auto=format,compress&w=10" sizes="100vw" data-optimize-img>
+```
+
+**Background Images**
+
+Background images work similarly, with three exceptions:
+
+- A `data-optimize-bg-img` attribute is required for the processing.
```

Comment:
Same as above... our filter could return an entire IMG tag, including this attribute.